src/components/home/What.jsx

```diff
@@ -0,0 +1,20 @@
+import Title from "@/components/Title";
+import Button from "@/components/Button";
+
+const What = () => {
+  return (
+    <div className="flex flex-col items-center justify-center h-screen">
+      <Title text="What We Do" />
+      <p class="text-center text-xl mx-10 my-5">
```

Comment:
What does the margining do? can you send a screenshot of the whole page of the component?

Reply:
I thought the text looked too close to the edge without the margin in the x-direction, but after making the changes you suggested and having the text not take up the whole page, it does not look like that anymore, so I removed the margin in the x-direction and now it looks like this: 
<img width="1123" alt="image" src="https://github.com/acm-ucr/isa-website/assets/102336530/8e792b20-2be6-4981-9097-ad56355adda4">

Reply:
i see, the text is still a little too close to the edge so i'd use width instead of margining for the paragraph tag. margining is typically used to add small spaces between elements since it pushes out of the container box.

for widths use fraction values.

Reply:
I changed the width so it now looks like this: 
<img width="1127" alt="image" src="https://github.com/acm-ucr/isa-website/assets/102336530/841cdc85-f904-42fe-958c-a56ecf3a6b1e">